fhir-server-test/pom.xml

```diff
@@ -124,7 +124,7 @@
         </dependency>
         <dependency>
             <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-nop</artifactId>
+            <artifactId>jcl-over-slf4j</artifactId>
```

Comment:
Why this change?

Reply:
There are no integration tests included.

Reply:
I was getting the SLF4J multiple bindings in classpath error when running unit tests. This fixes that problem.